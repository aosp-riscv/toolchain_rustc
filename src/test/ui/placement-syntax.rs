fn main() {
    let x = -5;
<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
    if x<-1 { //~ ERROR expected `{`, found `<-`
=======
    if x<-1 { //~ ERROR unexpected token: `<-`
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
        println!("ok");
    }
}
