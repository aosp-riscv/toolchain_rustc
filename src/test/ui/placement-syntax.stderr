<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
error: expected `{`, found `<-`
=======
error: unexpected token: `<-`
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
  --> $DIR/placement-syntax.rs:3:9
   |
LL |     if x<-1 {
<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
   |     --  ^^ expected `{`
   |     |
   |     this `if` statement has a condition, but no block
=======
   |         ^^
help: if you meant to write a comparison against a negative value, add a space in between `<` and `-`
   |
LL |     if x< -1 {
   |         ^^^
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)

error: aborting due to previous error

